Tests/test_PDB_MMCIFParser.py

```diff
@@ -323,10 +323,18 @@ def _run_point_mutation_tests(self, parser):
     def test_header(self):
         """Test if the parser populates header data."""
         parser = MMCIFParser()
+
+        structure = parser.get_structure("example", "PDB/a_structure.cif")
+        self.assertEqual("", structure.header["idcode"])
+        self.assertEqual("", structure.header["head"])
+        self.assertEqual("", structure.header["deposition_date"])
+        self.assertEqual("", structure.header["structure_method"])
+        self.assertEqual(0.0, structure.header["resolution"])
+
         structure = parser.get_structure("example", "PDB/1A8O.cif")
         self.assertEqual("1A8O", structure.header["idcode"])
         self.assertEqual("Viral protein", structure.header["head"])
-        self.assertIsNone(structure.header["deposition_date"])
+        self.assertEqual("", structure.header["deposition_date"])
```

Comment:
Should this be None instead?

Reply:
I don't think it should be None here because the other entries default to "" to be consistent with PDB parsing, but PDBParser generates a default date of 1909-01-08.  
- arguably both PDB and CIF parsers should set things that are missing to None, but that ship has probably sailed 
- I note there's no 'test we handle no header data sensibly' entry in the Tests/ set BTW
- I do not know what the relevance of that date is, but does not seem beneficial that it is definitely 'wrong' for any data here

So I don't know what the right thing is  :-)